fe/fe-core/src/main/java/org/apache/doris/nereids/parser/LogicalPlanBuilder.java

```diff
@@ -2470,11 +2450,11 @@ public LogicalPlan visitCreateTable(CreateTableContext ctx) {
                     keysType,
                     ctx.keys != null ? visitIdentifierList(ctx.keys) : ImmutableList.of(),
                     "",
-                    isAutoPartition,
-                    autoPartitionExpr.build(),
-                    partitionType,
-                    ctx.partitionKeys != null ? visitIdentifierList(ctx.partitionKeys) : null,
-                    ctx.partitions != null ? visitPartitionsDef(ctx.partitions) : null,
+                    partitionInfo.isAutoPartition(),
```

Comment:
if u aready merge all partition info into one class, u'd better update `CreateTableInfo`'s ctor to accept `PartitionTableInfo` as parameter